web-console/src/views/tasks-view.tsx

```diff
@@ -519,6 +520,16 @@ ORDER BY "rank" DESC, "created_time" DESC`);
               const previewCount = countBy(previewValues);
               return <span>{Object.keys(previewCount).sort().map(v => `${v} (${previewCount[v]})`).join(', ')}</span>;
             },
+            sortMethod: (status1: string, status2: string) => {
+              const statusRanking: any = this.statusRanking;
+              if (statusRanking[status1] > statusRanking[status2]) {
```

Comment:
A prettier way to describe this is `return statusRanking[status1] - statusRanking[status2]`